frontend/css/style.css

```diff
@@ -0,0 +1,40 @@
+:root {
+    --text-color: #0A0707;
+    --bg-color: #FDFCFC;
+    --text-correct-color: #177915;
+    --primary-color:#903634;
+    --secondary-color: #C99E9E;
+    --accent-color : #7DD0F3;
+    --card-bg-color: #E2E8F0;
+    --select-color : #7DD0F3;
+  }
+  
+  .dark-mode {
+    --text-color: #DBF6FD;
+    --bg-color: #181B1C;
+    --text-correct-color: #177915;
+    --primary-color:#9E092A;
+    --secondary-color: #434A5A;
+    --accent-color : #A5CAD7;
+    --card-bg-color: #202528;
+    --select-color : #7DD0F3;
+  }
+
+/**
+@media (prefers-color-scheme: dark) {
+    :root {
+    --text-color: #DBF6FD;
+    --bg-color: #181B1C;
+    --text-correct: #177915;
+    --primary-color:#9E092A;
+    --secondary-color: #434A5A;
+    --accent-color : #A5CAD7;
+    --card-bg-color: #202528;
+    --select-color : #7DD0F3;
+    }
+  }
+*/
+body {
+    color: var(--text);
```

Comment:
please update variable to match the new one :)

Reply:
Did not see that one 😃